packages/skill-lesson/lib/products.ts

```diff
@@ -24,6 +24,32 @@ const productsQuery = groq`*[_type == "pricing"][0] {
   }
 }`
 
+const allProductsQuery = groq`*[_type == "pricing"] {
+  title,
+  subtitle,
+  "products": products[]->{
+  "name": title,
+  productId,
+  description,
+  action,
+  image {
+    url,
+    alt
+  },
+  modules[]->{
+    title,
+    "image": image.asset->{url, alt},
+    state
+  },
+  features[]{
+    value
+  }
+  }
+}`
+
+export const getAllActiveProducts = async () =>
+  (await sanityClient.fetch(allProductsQuery)) || []
+
```

Comment:
was thinking about the "active" products, but didn't get very far

Reply:
`pricing` is the component we use to display the tiers in the home page, not necessarily all the active products. We already took care of `inactive` products, for the home page we could do a query for active `products` instead of `pricing`